specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-07-01/virtualMachineScaleSet.json

```diff
@@ -3644,6 +3650,37 @@
       },
       "description": "Describes an upgrade policy - automatic, manual, or rolling."
     },
+    "ResiliencyPolicy": {
+      "properties": {
+        "resilientVMCreationPolicy": {
+          "$ref": "#/definitions/ResilientVMCreationPolicy",
+          "description": "The configuration parameters used while performing resilient VM creation."
+        },
+        "resilientVMDeletionPolicy": {
+          "$ref": "#/definitions/ResilientVMDeletionPolicy",
+          "description": "The configuration parameters used while performing resilient VM deletion."
+        }
+      },
+      "description": "Describes an resiliency policy - resilientVMCreationPolicy or resilientVMDeletionPolicy."
```

Comment:
Since we can create a resource with both ResilientVMCreation and ResilientVMDeletion policy
can we update this description to reflect this?

'resilientVMCreationPolicy and/or resilientVMDeletionPolicy.'